testkomodo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ENV=testenv
+rm -rf testenv
+mkdir $ENV
+python -m virtualenv --system-site-packages $ENV
+source $ENV/bin/activate
+python -m pip install mock
```

Comment:
I think we should have a separate `test_requirements.txt` file, and rather install that instead. However, that might be a separate issue.

Reply:
Feel free to make the issue ;)

Reply:
https://github.com/equinor/ert/issues/485